cpp/dolfinx/io/cells.cpp

```diff
@@ -152,120 +156,63 @@ std::vector<std::uint8_t> io::cells::dolfin_to_vtk(mesh::CellType type,
   }
 }
 //-----------------------------------------------------------------------------
-std::vector<std::uint8_t> io::cells::vtk_to_tp(mesh::CellType type,
-                                               int num_nodes)
-{
-  const std::vector<std::uint8_t> reversed
-      = io::cells::dolfin_to_vtk(type, num_nodes);
-  switch (type)
-  {
-  case mesh::CellType::quadrilateral:
-  {
-    std::vector<std::uint8_t> perm(num_nodes);
-    for (int i = 0; i < num_nodes; ++i)
-      perm[reversed[i]] = i;
-    return perm;
-  }
-  case mesh::CellType::hexahedron:
-  {
-    std::vector<std::uint8_t> perm(num_nodes);
-    for (int i = 0; i < num_nodes; ++i)
-      perm[reversed[i]] = i;
-    return perm;
-  }
-  default:
-    throw std::runtime_error("Simplicies can be expressed as TensorProduct");
-  }
-}
-//-----------------------------------------------------------------------------
-std::vector<std::uint8_t> io::cells::lex_to_tp(mesh::CellType type,
-                                               int num_nodes)
-{
-  switch (type)
-  {
-  case mesh::CellType::quadrilateral:
-  {
-    assert((std::sqrt(num_nodes) - std::floor(std::sqrt(num_nodes))) == 0);
-    // Number of nodes in each direction
-    const int n = sqrt(num_nodes);
+// std::vector<std::uint8_t> io::cells::lex_to_tp(mesh::CellType type,
```

Comment:
Do we need this commented code here, even for @jorgensd ? Git has a decent history.